compiler/src/dotty/tools/dotc/decompiler/DecompilationPrinter.scala

```diff
@@ -24,8 +26,9 @@ class DecompilationPrinter extends Phase {
       var os: OutputStream = null
       var ps: PrintStream = null
       try {
+        implicit val codec = Codec.UTF8
         os = File(outputDir.fileNamed("decompiled.scala").path).outputStream(append = true)
```

Comment:
I would not introduce an implicit in scope but pass the implicit explicitly:
```suggestion
        os = File(outputDir.fileNamed("decompiled.scala").path)(Codec.UTF8).outputStream(append = true)
```